app/controllers/memes_controller.rb

```diff
@@ -48,6 +58,9 @@ def update
   end
 
   def secret
+    if current_user.admin
+
+    end
```

Comment:
What is this method doing? If nothing yet, delete it and add it back when you're actually using it